openapi.yaml

```diff
@@ -3051,29 +3051,27 @@ paths:
         For more detailed information on a particular Database instance, make a request to the `instance_uri` for that instance.
       responses:
         '200':
-          description: Returns information for a single Managed Database engine.
+          description: Returns information for a single Managed Database version.
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/DatabaseEngine'
+                $ref: '#/components/schemas/DatabaseVersion'
         default:
           $ref: '#/components/responses/ErrorResponse'
       x-code-samples:
-        - lang: Shell
-          source: |
-            curl https://api.linode.com/v4beta/databases/engines//mysql/5.7.30
-        - lang: CLI
-          source: |
-            linode-cli databases engine-view mysql/5.7.30
+      - lang: Shell
+        source: >
+          curl https://api.linode.com/v4beta/databases/versions/mysql/5.7.30
+      - lang: CLI
+        source: >
+          linode-cli databases version-view mysql/5.7.30
   /databases/instances:
     x-linode-cli-command: databases
     get:
       tags:
         - Databases
       summary: Managed Databases List
       operationId: getDatabasesInstances
-      servers:
```

Comment:
any reason this is removed? should only be accessible at v4beta, and this designates the command as in beta